src/commands/deploy/genezio.ts

```diff
@@ -751,6 +752,11 @@ export async function deployFrontend(
 
     frontend.subdomain = options.subdomain || frontend.subdomain;
 
+    // If authentication is enabled in the project - attempt to set a default password reset URL
+    if (configuration.services?.authentication) {
+        await setDefaultPasswordRedirectUrl(name, stage, frontend.subdomain || "");
+    }
```

Comment:
What if there is no backend deployed? This will fail because it will not find any project with this name. I would move this after creating the frontend so we could be sure that the environment exists.